src/Composition/object3D.jl

```diff
@@ -178,7 +178,8 @@ mutable struct Object3D{F} <: Modia3D.AbstractObject3D
 
         if !isnothing(parent)
             # with parent -> call constructor 3
-            if !isnothing(rotation) && (typeof(rotation) != Frames.RotationMatrix)
+            if !isnothing(rotation)
+                rotation = Modia3D.convertAndStripUnit(SVector{3,F}, u"m"  , rotation)
```

Comment:
"m" is wrong for angles, should be "rad"

Reply:
Thanks.